sidecar-injector/bin/publish

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash -e
```

Comment:
Don't we need to build the image first?

Reply:
yup, it's happening below

Reply:
i didn't see any value to having an independent `build` step